packages/sfui/shared/scrollable/scrollable.ts

```diff
@@ -133,8 +141,53 @@ export default class Scrollable {
     }
   }
 
-  private onMouseUp() {
+  private onMouseUp(event: MouseEvent) {
+    const { container, options } = this;
```

Comment:
whole logic is moved to mouse up because previous version was not reliable in terms of dragging, e.g situation when user holding mouse or touch pad dragging left, right and then release mouse, we had no idea in which direction we should move page. Of course we can calculate it but IMHO too much effort for now.